flowetl/tests/integration/conftest.py

```diff
@@ -9,9 +9,11 @@
 import os
 import random
 import string
+import tarfile
```

Comment:
What for?

Reply:
Once there was a hairy test fixture that pushed files to a running container, but it's gone now. Will drop.